ArchiSteamFarm/ArchiSteamFarm.csproj

```diff
@@ -30,6 +30,7 @@
 	</ItemGroup>
 
 	<ItemGroup Condition="'$(TargetFramework)' != 'net48'">
+		<PackageReference Include="Microsoft.Win32.SystemEvents" />
```

Comment:
We can probably use `RuntimeIdentifier` as a condition for this dependency. If not, we can definitely use `ASFVariant`, which is always available here (we currently use it in `Directory.Build.props` to define some constants).

Alternatively we could leave the dependency in all builds and hope trimming has gotten good enough to remove it, once we have those `#if` statements in the code.

Reply:
Not critical for this PR.